include/supl_os_client.rst

```diff
@@ -0,0 +1,35 @@
+.. _supl_client:
+
+SUPL client
```

Comment:
How should this be called? We call it supllib in the bin folder and supl_os_client in the include folder ... Should this be "SUPL OS client" instead?

Reply:
I think SUPL client is ok. SUPLlib is the specific library component that provides the SUPL Client functionality, supl_os_client refer to the OS integration code for using this client with NCS. But all of this is parts that provides SUPL Client functionality to NCS.

Reply:
And it won't be confusing for users that the header file has a different name than the library folder?

Reply:
I agree with Ruth, it would be best that the two things had the same name.